interface/portraits/portraits_kvaldir_sprites.gfx

```diff
@@ -40,530 +26,33 @@ spriteTypes = {
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_nose_1.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_nose_midage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_nose_2.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-	
-	spriteType = {
-		name = "GFX_kvaldir_male_nose_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_nose_3.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-
 	spriteType = {
 		name = "GFX_kvaldir_male_eyes"
 		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_eyes_1.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_eyes_midage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_eyes_2.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_eyes_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_eyes_3.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_eyes2"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_eyes2.dds"
-		noOfFrames = 13
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_headgear_mid"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_headgear_mid.dds"
-		noOfFrames = 12
-		norefcount = yes
-	}
-
-
-	spriteType = {
-		name = "GFX_kvaldir_male_ear"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_ear_1.dds"
-		noOfFrames = 5
+		noOfFrames = 6
 		norefcount = yes
 	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_ear_midage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_ear_2.dds"
-		noOfFrames = 5
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_ear_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_ear_3.dds"
-		noOfFrames = 5
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_beard"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_beard_1.dds"
-		noOfFrames = 10
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_beard_midage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_beard_2.dds"
-		noOfFrames = 10
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_beard_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_beard_3.dds"
-		noOfFrames = 10
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_hair"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_hair_1.dds"
-		noOfFrames = 15
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_hair_midage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_hair_2.dds"
-		noOfFrames = 15
-		norefcount = yes
-	}
-
-	spriteType = {
-		name = "GFX_kvaldir_male_hair_oldage"
-		texturefile = "gfx\\characters\\kvaldir_male\\kvaldir_male_hair_3.dds"
-		noOfFrames = 15
-		norefcount = yes
-	}
-
+	
 	spriteType = {
 		name = "GFX_kvaldir_male_clothes"
```

Comment:
It's headgear, not clothes. Headgears can hide hairs, clothes - not.
So you may notice his hair on this screenshot.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/77094878-8ae84200-6a26-11ea-813d-33df86872fca.png)

</details>

Reply:
fixed

Reply:
Not fixed, because hairs are located on the eyes layer.

Reply:
they cant be on hair layers because it removes colors

Reply:
Why normal and frost vrykul hairs work?

Reply:
because normal vrykul use normal hair colors

Reply:
and frost vrykul hair looks bad imo

Reply:
because the colors arent as good as blizz textures for weird colors

Reply:
@sejtalk I think this color is pretty good: `{ 0 0 0 } {  100 120 50 } { 255 255 255 }`
P.S. Can find more colors: yellower, greener, etc.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/77201575-28ff0980-6b06-11ea-84e8-bda276724f81.png)

</details>